source/includes/code-examples/DeleteOneSync.cs

```diff
@@ -0,0 +1,70 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOneSync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Delete a document using builders
+        WriteLine("Deleting a document with builders...");
+        var result = DeleteARestaurantBuilderSync();
+
+        WriteLine($"Deleted documents: {result.DeletedCount}");
+        // Extra space for console readability 
+        WriteLine();
```

Comment:
These lines aren't needed since we aren't adding any other output below the last WriteLine